### ISSUE TITLE
Can this be used with CRA boilerplate??

### ISSUE THREAD

**Issue by Shubham-DK** [davidhu2000/react-spinners#182]
I am wondering can this be directly used with Create React app Boiler plate, as you mentioned to add plugin in .babelrc

**Comment by davidhu2000** [comment]
Of course, if you just run `create-react-app`, then `npm install react-spinners`. It'll be added to the app.

**Comment by Shubham-DK** [comment]
So no need to modify anything in .babelrc ?

**Comment by nickojs** [comment]
Hey @davidhu2000, I don't get it. When importing a spinner, the following error is thrown:

`./node_modules/react-spinners/ScaleLoader.js
Module not found: Can't resolve '/frontend/node_modules/babel-loader/lib/index.js' in '/frontend'
`

As 'frontend' being the folder where I created a react app with CRA. Do I have to eject and edit the CRA's babel config file? Also, I tried adding a babelrc file with the instructions you provided, but also faild 

Anyway, great project bro. Looking forward to use it on my projects!
